scss/mixins/_color-mode.scss

```diff
@@ -13,9 +13,18 @@
       }
     }
   } @else {
-    [data-bs-theme="#{$mode}"] {
-      @content;
+    // Boosted mod
```

Comment:
Check if it is really useful for this PR. If not yet, let's drop it and wait for https://github.com/twbs/bootstrap/pull/39295